pkg/osquery/runtime/osqueryinstance.go

```diff
@@ -375,14 +375,21 @@ func (i *OsqueryInstance) Launch() error {
 		return fmt.Errorf("starting osqueryd process: %w", err)
 	}
 
-	stats, err := history.NewInstance(i.registrationId, i.runId)
-	if err != nil {
+	osqHistory := i.knapsack.OsqueryHistory()
```

Comment:
Why can't we use `i.history` here instead?

Reply:
oop because I was waffling on whether to keep i.history at all and then never updated, I will get this cleaned up next. thank you!